### ISSUE TITLE
/analytics/model-compare presents hand-written formulas as four model predictions

### ISSUE THREAD

**Issue by Evgeniy2001Poluhin** [Evgeniy2001Poluhin/-SORA.Earth#323]
Found while working on #320. Not fixed there: this route does not touch the neural network, or any model.

## What the code does

`POST /api/v1/analytics/model-compare` — `app/api/analytics.py:131-151`:

```python
rf_p  = max(0.0, min(1.0, 0.30 * base + 0.30 * soc + 0.25 * dur + 0.15 * eco))
xgb_p = max(0.0, min(1.0, 0.34 * base + 0.28 * soc + 0.23 * dur + 0.15 * eco))
nn_p  = max(0.0, min(1.0, 0.28 * base + 0.34 * soc + 0.23 * dur + 0.15 * eco))
ens_p = max(0.0, min(1.0, (rf_p + xgb_p + nn_p) / 3.0))
```

Four hand-written linear formulas over `co2_reduction`, `social_impact`, `duration_months` and `budget`, with coefficients that appear nowhere else. No model object is imported or called. The results are returned labelled `RandomForest`, `XGBoost`, `NeuralNet` and `StackingEnsemble`, with a `best_model` picked among them and `threshold` hard-coded to `0.5` — while the real threshold (`best_threshold`) is 0.63 in the current seed.

## Who sees it

`app/static/app.js:297` and `:344` call it, and `app/main.py` mounts `app/static` at `/static`. The React SPA does not call it. `tests/test_analytics.py:66-72` asserts all four keys are present, which the formulas guarantee.

## Why it matters

A comparison of models that runs no model reads, on a screen, exactly like one that does. It is the shape recorded after the mock ESG scores: a working page that cannot be wrong about the thing it claims to show.

## Direction, not decided

Either compute through the loaded models — `_base_probabilities` in `app/api/predict.py`, introduced by #322, does exactly that and omits a model that is not loaded — or remove the route and its callers in `app/static/app.js`. Presenting formulas under model names is not one of the options.
